app/controllers/api/subcollections/container_images.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module Subcollections
+    module ContainerImages
+      def container_images_scan_resource(provider_id, type = nil, image_id = nil, _payload = nil)
+        raise BadRequestError, "Must specify an id for scanning a #{provider_id} resource" unless type
```

Comment:
Can you add a test for this? Honestly, I'd be surprised if you could be routed here without a provider id

Reply:
you're right; I picked up this line when looking at another endpoint as a template (the VM scanning one). since the path includes a provider ID, this should never be reachable code